queue/src/main/java/org/killbill/bus/InMemoryPersistentBus.java

```diff
@@ -109,20 +109,25 @@ public void postFromTransaction(final BusEvent event, final Connection connectio
 
     @Override
     public boolean initQueue() {
-        return false;
-    }
-
-
-    @Override
-    public boolean startQueue() {
         if (isInitialized.compareAndSet(false, true)) {
```

Comment:
Somehow orthogonal to this change, we could add the check for the `isProcessingOff` -- similarly to `DefaultPersistentBus#initQueue `

Reply:
Fixed in e098b7d4b72b6318685826c956fc409ae05ca21d.